src/validator/validator.ts

```diff
@@ -108,11 +108,11 @@ export const validator = <
         const form: BodyData<{ all: true }> = {}
         formData.forEach((value, key) => {
           if (key.endsWith('[]')) {
-            if (form[key] === undefined) {
-              form[key] = [value]
-            } else if (Array.isArray(form[key])) {
-              ;(form[key] as unknown[]).push(value)
-            }
+            ;((form[key] ??= []) as unknown[]).push(value)
+          } else if (Array.isArray(form[key])) {
+            ;(form[key] as unknown[]).push(value)
```

Comment:
@nicksrandall Can you write the test to cover this line?

Reply:
Okay!

Reply:
I *think* it might be better but the coverage action is failing in CI so I'm not sure

Reply:
@nicksrandall 

The CI seemed to fail because of accidentally. Restared, now it passed.